### PR TITLE
Consolitation of 2FA messages #103

### PR DESCRIPTION
**Pull request by maxxer**
| Q             | A
| ------------- | ---
| Fixed issues  | #103

### CODE REVIEW

**Comment by tonydspaniard** [comment]
Hi @maxxer, thank you very much for your collaboration and the user of your free time to make this project better. 

Nevertheless, when updating the messages, we have to run the `./vendor/yiisoft/yii2/yii message ./src/User/resources/i18n/config.php` command to update the messages.

File: src/User/Form/LoginForm.php [hunk 1]
```diff
@@ -73,7 +73,7 @@ public function attributeLabels()
             'login' => Yii::t('usuario', 'Login'),
             'password' => Yii::t('usuario', 'Password'),
             'rememberMe' => Yii::t('usuario', 'Remember me next time'),
-            'twoFactorAuthenticationCode' => Yii::t('usuario', 'Two-factor authentication code')
+            'twoFactorAuthenticationCode' => Yii::t('usuario', '2FA code')
```
Comment (tonydspaniard) [inline src/User/Form/LoginForm.php]: I am not sure if 2FA is more descriptive than Two-Factor|Two Factor authentication.

Comment (tonydspaniard) [reply src/User/Form/LoginForm.php]: I know that I have agreed to it, but... not seems to brief. What do you think? We need to think as an external user.

Comment (maxxer) [reply src/User/Form/LoginForm.php]: I could agree with you, but generally I think that people who wants to enable 2FA knows what it is and have a bit of knowledge about its acronym. Also in the account settings form I've left the extended and compact strings together, so ppl can associate 2FA to two factor auth.

But if you prefer I can change the label. Is it just about that or elsewhere?

Comment (tonydspaniard) [reply src/User/Form/LoginForm.php]: Is everywhere, I think two factor authentication is more descriptive not for developers but for end users. Not many people know (in our case we always have to explain what it is and how it works on all our panels), so the acronym even though is clearly a plus for developers I am not really sure about end users.

Comment (maxxer) [reply src/User/Form/LoginForm.php]: made another commit

**Comment by tonydspaniard** [comment]
Thank you @maxxer I'll review asap

**Comment by tonydspaniard** [comment]
Quick update: @maxxer I didn't forget, I just do not have much time this week. I hope I can fully review it this weekend. Again, thanks for your time and efforts to make this module better for all of us using Yii.

**Pull request merged**